explorer/interpreter/type_checker.cpp

```diff
@@ -1029,9 +1029,10 @@ auto TypeChecker::ImplicitlyConvert(std::string_view context,
     destination = destination_constraint;
 
     if (trace_stream_->is_enabled()) {
-      *trace_stream_ << "converting type " << *converted_value
-                     << " to constraint " << *destination_constraint << " for "
-                     << context << " in scope " << impl_scope << "\n";
+      *trace_stream_ << "converting type `" << *converted_value
+                     << "` to constraint `" << *destination_constraint
+                     << "` for `" << context << "` in scope " << impl_scope
```

Comment:
`context` here is a string that's intended to be interpolated into the trace message; I think we shouldn't put backticks around it.
```suggestion
                     << "` for " << context << " in scope " << impl_scope
```